src/transformStateWithClones.js

```diff
@@ -7,7 +7,38 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const CLONE_STATE = { ...state };
+  const HISTORY_STATE = [];
+
+  for (let i = 0; i < actions.length; i++) {
+    const CURRENT_ACTION = actions[i];
+    const ACTION_TYPE = CURRENT_ACTION.type;
```

Comment:
All these variables are not constants, so they should be in camelCase. Constants are variables that are static and never change.